operators/index.js

```diff
@@ -6,31 +6,57 @@ const {
   seekChannel,
   seekRoot,
   seekPrivate,
+  seekVoteLink,
 } = require('../seekers')
 const { equal } = jitdbOperators
 
 function key(value) {
-  return equal(seekKey, value, { indexType: 'key', prefix: 32 })
+  return equal(seekKey, value, {
+    prefix: 32,
+    indexType: 'key',
+  })
 }
 
 function type(value) {
-  return equal(seekType, value, { indexType: 'value_content_type' })
+  return equal(seekType, value, {
+    indexType: 'value_content_type',
+  })
 }
 
 function author(value) {
-  return equal(seekAuthor, value, { indexType: 'value_author' })
+  return equal(seekAuthor, value, {
+    indexType: 'value_author',
```

Comment:
I'm curious what would the performance look like if we made all these prefix indexes! `author`, `type` and `channel`.

That's something to benchmark soon

Reply:
Yes, I think it def. makes sense for channels. Author would be interesting to contrast with using a deferred on the the level index to see. For type if we can get the sorted lookup in there then yes, otherwise I'm leaning towards a specific index for that because its so heavily used.